crates/shared/src/oneinch_api.rs

```diff
@@ -179,12 +181,33 @@ pub struct SwapQuery {
     pub main_route_parts: Option<Amount<1, 50>>,
     /// Limit maximum number of parts each main route part can be split into.
     pub parts: Option<Amount<1, 100>>,
+    /// Receiver of destination currency. default: fromAddress
+    pub dest_receiver: Option<H160>,
```

Comment:
Nit: Some variables will never be used, e.g. dest_receiver will always be the fromAddress. 
Hence, in order to keep code shorter, it might make sense to exclude these struct fields.
(I really don't have a strong opinion here, and feel free to keep it as it is.)

Reply:
Since this is implemented based on the 1Inch API documentation, I think its fine to include it here. No strong preference either way.